web/src/lib/components/pages/record/record.css

```diff
@@ -252,6 +252,17 @@
     border: solid 0.1rem;
 }
 
+#profile-form-container {
```

Comment:
the CSS should go to a separate file IMO

Reply:
done